src/nncf/quantization/algorithms/weight_compression/scale_estimation.py

```diff
@@ -366,6 +372,14 @@ def calculate_quantization_params(
         if zp is not None and config.group_size == -1:
             zp = fns.squeeze(zp, axis=1)
 
+        if was_transposed:
```

Comment:
suggest covering this fix in a template test as it affects common part.

Reply:
It is already covered in `TemplateWeightCompression`. The tests do not pass without this fix.